repondeur/zam_repondeur/models/article.py

```diff
@@ -199,6 +199,22 @@ def create(
         DBSession.add(article)
         return article
 
+    @property
+    def previous_article(self) -> Optional["Article"]:
+        sorted_articles: List[Article] = sorted(self.lecture.articles)
+        previous_index = sorted_articles.index(self) - 1
+        if previous_index < 0:
+            return None
+        return sorted_articles[previous_index]
+
+    @property
+    def next_article(self) -> Optional["Article"]:
+        sorted_articles: List[Article] = sorted(self.lecture.articles)
+        next_index = sorted_articles.index(self) + 1
+        if next_index >= len(sorted_articles):
+            return None
+        return sorted_articles[next_index]
+
```

Comment:
Maybe we should skip / filter out intersticial articles. We don't show them in the list as we don't need to edit info about them, so it's surprising to land on one when navigating to the next article.

Reply:
It is done if you want to take a look